oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -37,8 +37,7 @@
 import com.google.api.client.util.Preconditions;
 import com.google.auth.http.HttpTransportFactory;
 
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
We don't use *'s
https://google.github.io/styleguide/javaguide.html

Reply:
fixed